octoprint_mrbeam/static/js/app/view-models/working-area.js

```diff
@@ -3103,6 +3103,9 @@ $(function () {
                 "change",
                 throttle((event) => self._qt_currentQuickTextUpdate(event), 200)
             );
+
+            // todo change to using LASER_CUTTER_MODE_NAME
```

Comment:
just for consistency as it is written this way everywhere
```suggestion
            // TODO: change to using LASER_CUTTER_MODE_NAME
```